miden-lib/asm/kernels/transaction/main.masm

```diff
@@ -68,6 +68,9 @@ proc.main.1
     exec.prologue::prepare_transaction
     # => []
 
+    push.0 drop
```

Comment:
Let's add a simple comment to these lines everywhere to note why we are doing `push.0 drop` (we have these comments in other places).